src/demos/block/basic.html

```diff
@@ -90,6 +90,18 @@ <h3>Header + content (collapsible)</h3>
               <img alt="demo" src="https://placeimg.com/640/480/nature" />
             </calcite-block-section>
           </calcite-block>
+
+          <h3>With sections</h3>
```

Comment:
Should this be  `<h2>Header + content (sections w/ switch)</h2>`?

Reply:
oh yeah.

Reply:
Updated.